client/resultPage.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="resultPage.css" />
+		<title>Google Results</title>
+		<script defer src="results.js"></script>
+	</head>
+	<body>
+		<section id="logo"><img id="googleLogo" src="./images/Google_2015_logo.svg.png" /></section>
+
+		<section id="searchBar">
+			<div class="bar">
+				<input class="searchbar" type="text" title="Search" />
+				<a href="#">
+					<img
+						class="voice"
+						src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e8/Google_mic.svg/716px-Google_mic.svg.png"
+						title="Search by Voice"
+				/></a>
+			</div>
+		</section>
+
+		<section id="signIn">
+			<ul>
+				<li>
+					<a class="links" href="#user"
+						><button class="signbutton" type="button">Sign in</button></a
+					>
+				</li>
+				<li>
+					<a href="#grid"
+						><img
+							class="grid"
+							src="https://cdn3.iconfinder.com/data/icons/navigation-and-settings/24/Material_icons-01-11-512.png"
+							title="Google apps"
+					/></a>
+				</li>
+				<li><a href="#images">Images</a></li>
+				<li><a href="#gmail">Gmail</a></li>
+			</ul>
+		</section>
+
+		<!-- <article> -->
+		<aside></aside>
+		<div id="filler"></div>
+		<main></main>
+		<aside></aside>
+		<!-- </article> -->
+		<footer></footer>
+	</body>
+</html>
```

Comment:
similar comments to the index page
indentation a bit mixed
no favicon
alt tags missing